domain/service/servicedetails_store.go

```diff
@@ -119,12 +122,10 @@ func (s *storeImpl) GetServiceDetails(ctx datastore.Context, serviceID string) (
 	searchRequest := newServiceDetailsElasticRequest(map[string]interface{}{
 		"query": map[string]interface{}{
 			"ids": map[string]interface{}{
-				"values": []string{id},
+				"values": []string{fmt.Sprintf("%s-%s", id, kind)},
```

Comment:
Similar comment from connection.go, the format shouldn't leak everywhere. The Key type should generated the ID.

Reply:
It was a necessary decision after ES removed mapping types. Other approaches require more changes in the code. 
I've included type to ID to avoid collisions, because of now all records are in one document `controlplane`

Reply:
I understand why the IDs changed to have the type. I just meant the string format is leaked everywhere it is used.  I was thinking something like this so that the format doesn't have to be known by every package.

```
func (k *key) DocumentID() string {
	return fmt.Sprintf("%s-%s", k.kind, k.id)
}
```

Reply:
transferred to the separate function didn't saw the comment above at the first time